src/less/success-message/base/success-message.less

```diff
@@ -40,6 +20,25 @@
     padding: 0 9%;
     text-align: center;
     width: 100%;
+
+    .btn--secondary,
+    .fake-btn--secondary {
+        border-color: @success-message-button-border-color;
+        color: @success-message-button-color;
+        cursor: pointer;
+        margin-top: 28px;
```

Comment:
It might be better (as per what Ian was saying for overriding styles) to maybe move this to the container above so that has a bottom margin of 28px.

Reply:
The button is optional, so if we added it to the container above it would add unneeded space when the button is missing.